libpod/copy_data.go

```diff
@@ -34,6 +36,28 @@ const (
 	OCIv1ImageManifest = v1.MediaTypeImageManifest
 )
 
+// Data handles the data used when inspecting a container
+// nolint
+type Data struct {
+	ID           string
+	Tags         []string
+	Digests      []string
+	Digest       digest.Digest
```

Comment:
Another Digest/Digests in a struct.  I'd love for one of these to be renamed if at all possible.

Reply:
renamed to ManifestDigest. Fixed